packages/react/src/pages/planning-task-list/planning-task-list.tsx

```diff
@@ -19,53 +27,90 @@ const listsData = ['LIST', 'KANBAN BOARD', 'GANTT'];
 
 export const PlanningTaskList = () => {
   const gridRef = useRef<DataGrid>(null);
+  const kanbanRef = useRef<Sortable>(null);
+  const ganttRef = useRef<Gantt>(null);
+
+  const navigate = useNavigate();
 
   const [list, setList] = useState(listsData[0]);
```

Comment:
Replace for increace code readability
const [listItem, kanbanItem, ganttItem] = listData;
const [list, setList] = useState(listItem);